1-weekly-mission-week2-main/sign.js

```diff
@@ -0,0 +1,64 @@
+const email_input = document.querySelector('#email_input');
+const pw_input = document.querySelector('#pw_input');
+const error_email = document.querySelector('#error_email');
+const error_pw = document.querySelector('#error_pw');
+const eye = document.querySelector('.eye');
+const email_regex = /^[a-zA-Z0-9._-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,4}$/;
+
+function inEmail(){
+  return email_regex.test(email);
+}
+
+//email
+function EmailInput(){
+  if (email_input.value.trim() === ""){
+    error_email.innerText = "이메일을 입력해 주세요.";
+  } else if(!inEmail(email_input.value.trim())){
+    error_email.innerText = "올바른 이메일 주소가 아닙니다.";
+  } else {
+    error_email.style.display = "none";
+  }
```

Comment:
```suggestion
  if (email_input.value.trim() === ""){
    error_email.innerText = "이메일을 입력해 주세요.";
    return;
  }
  if(!inEmail(email_input.value.trim())){
    error_email.innerText = "올바른 이메일 주소가 아닙니다.";
    return;
  } 
  error_email.style.display = "none";
```
return문을 활용하면 가독성이 더 좋아질 수 있을것 같습니다!